src/sentry/static/sentry/app/views/performance/landing/display/singleAxisChart.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {Location} from 'history';
+
+import {Organization} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+
+import _Footer from '../../charts/footer';
+import {AxisOption} from '../../data';
+import DurationChart from '../chart/durationChart';
+import HistogramChart from '../chart/histogramChart';
+
+type DisplayProps = {
+  location: Location;
+  organization: Organization;
+  eventView: EventView;
+  axis: AxisOption;
+  onFilterChange: (minValue: number, maxValue: number) => void;
```

Comment:
Can we add a comment here mentioning that this is only used for distribution graphs?